README.md

```diff
@@ -1,73 +1,63 @@
-targetcli
-=========
+# iscsi-target
 
-Targetcli target iSCSI role which takes care of installing targetcli, enabling it on boot and configuring it based on the provided variable from ansible.
+iSCSI target role that installs and uses targetcli.
 
-NOTE: Modules used by this module were separated into role OndrejHome.targetcli-modules
+## Requirements
```

Comment:
-------------		 +Also, this role is not creating any disks/partitions/LVs. Therefore it is expected that they are already present on machine or created by some other role. 
link to role that creates disks and paritions not in require ments mbut in the description